src/errorUtils.js

```diff
@@ -0,0 +1,126 @@
+export function error2Msg(errorCode, str) {
+  // let 501_Error_Code = "Text should start with Department name";
+  switch (errorCode) {
+    case 501:
+      return {
+        message: "Text should start with Department name (with Letter only)",
+        error: true
```

Comment:
All are errors, so what is the use of returning error boolean here for every case? Only return message string, no boolean error